Security controls > Security logging & events

iamnotacake opened the pull request:
* Add examples of what Acra can log
* Page for _Programmatic reactions_
* Pages for _Security events_ and _SIEM/SOC integration_ (mostly empty)

Lagovas commented on acra/security-controls/security-logging-and-events/_index.md:
```diff
@@ -7,6 +7,31 @@ bookCollapseSection: true
+
+Keystore events:
+* Reading master key from environment or KMS (i.e. HashiCorp Vault), possible errors
+* Loading other\* keys from filesystem or key DB (i.e. Redis), possible errors
```
here we should admit that each request to keystore is the same as authorization or data access request.  Acra logs client/zone IDs that tried to get access.

Lagovas commented on acra/security-controls/security-logging-and-events/_index.md:
```diff
@@ -7,6 +7,31 @@ bookCollapseSection: true
+
+\* — the ones needed for encryption/decryption, HMAC, audit log, basically everythings except the master key
+
+Issues with a database schema:
```
or `encryptor_config` that strongly depends on database schema

Lagovas commented on acra/security-controls/security-logging-and-events/_index.md:
```diff
@@ -20,6 +45,12 @@ bookCollapseSection: true
+* Poison record access
+
+  Access to data inside database that was not meant to be accessed at all.
+  These poison records can be inserted to catch situations like malicious SQL client attempting to dump the whole database.
```
not only SQL clients. AcraTranslator supports poison records too. So poison records can be catched on data not only from database, but from any storage where it is saved in case with AT>

Lagovas commented on acra/security-controls/security-logging-and-events/programmatic_reactions.md:
```diff
@@ -0,0 +1,12 @@
+Performing certain actions (such as launching scripts) in certain situations.
+
+This currently includes [intrusion detection]({{< ref "acra/security-controls/intrusion-detection/_index.md" >}})
+and allows to stop AcraServer or launch a script upon detecting abnormal activity.
```
```suggestion
and allows to stop AcraServer/AcraTranslator or launch a script upon detecting abnormal activity.
```

iamnotacake commented:
> don't forget to fix `weight`

Added weights to all pages inside _Security controls_ (`acra/security-controls/*/_index.md`) to match the order from screenshot.

Also, removed duplicate pages for _SIEM/SOC integration_ & _Programmatic reactions_, they are now only in _Security controls_
